pkgs/development/python-modules/moderngl/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, libGLU_combined
+, libX11
+}:
+
+buildPythonPackage rec {
+  pname = "moderngl";
+  version = "5.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0x8xblc3zybp7jw9cscpm4r5pmmilj9l4yi1rkxyf0y80kchlxq4";
+  };
+
+  disabled = !isPy3k;
+
+  propagatedBuildInputs = [ libGLU_combined libX11 ];
```

Comment:
Why are these propagated?

Reply:
Used https://github.com/NixOS/nixpkgs/blob/f4811789605ee3fb1c069fd6c569aae32cd27d1e/pkgs/development/python-modules/pyopengl/default.nix as a template. The dependencies probably don't to be propagated. Not propagated would be `buildInputs`?